src/core/agent/services/multiSigService.ts

```diff
@@ -693,6 +694,26 @@ class MultiSigService extends AgentService {
       .exchanges()
       .send(name, "multisig", aid, route, payload, embeds, recp);
   }
+
+  @OnlineOnly
+  async isMultisigInitiator(multisigId: string): Promise<boolean> {
```

Comment:
Actually, there are 2 conditions:
1. We have the multisig in our wallet (either we are initiator, or we are receiver and accepted)
2. We already have a pending multi-sig notification (e.g. we got 2 in a row without accepting/declining the first)

So in this function, we just need to check if we have the multi-sig identifier in our wallet.